app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PPOCardView.kt

```diff
@@ -0,0 +1,616 @@
+    }
+}
+
+@Composable
+fun FixPaymentButtonView(onFixPaymentClicked: () -> Unit) {
+    // TODO: Replace with translated string
+    KSSecondaryRedButton(
+        modifier = Modifier.padding(dimensions.paddingMediumSmall),
+        onClickAction = { onFixPaymentClicked.invoke() },
+        text = "Fix Payment",
+        isEnabled = true,
+        textStyle = typography.buttonText
+    )
+}
+
+@Composable
+fun AuthenticateCardAlertsView(daysRemaining: Int = -1) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = dimensions.paddingMediumSmall, start = dimensions.paddingMediumSmall)
+    ) {
+        KSCoralBadge(
+            leadingIcon = {
+                // TODO: Replace with translated string
+                Image(
+                    modifier = Modifier.padding(end = dimensions.paddingXSmall),
+                    imageVector = ImageVector.vectorResource(id = R.drawable.ic_icon_alert),
+                    contentDescription = "Card needs authentication",
+                    colorFilter = ColorFilter.tint(colors.textAccentRedBold)
+                )
+            },
+            // TODO: Replace with translated string
+            text = "Card needs authentication",
+            textColor = colors.textAccentRedBold
+        )
+
+        if (daysRemaining > 0) {
+            Spacer(modifier = Modifier.height(dimensions.paddingSmall))
+
+            KSCoralBadge(
+                leadingIcon = {
+                    // TODO: Replace with translated string
+                    Image(
+                        modifier = Modifier
+                            .padding(end = dimensions.paddingXSmall)
+                            .size(dimensions.alertIconSize),
+                        imageVector = ImageVector.vectorResource(id = R.drawable.ic_clock),
+                        contentDescription = "Pledge will be dropped in $daysRemaining days",
+                        colorFilter = ColorFilter.tint(colors.textAccentRedBold)
+                    )
+                },
+                // TODO: Replace with translated string
+                text = "Pledge will be dropped in $daysRemaining days",
+                textColor = colors.textAccentRedBold
+            )
+        }
+    }
+}
+
+@Composable
+fun PaymentFixedButtonView(onFixPaymentClicked: () -> Unit) {
+    // TODO: Replace with translated string
+    KSSecondaryRedButton(
+        modifier = Modifier.padding(dimensions.paddingMediumSmall),
+        leadingIcon = {
+            Image(
+                modifier = Modifier
+                    .padding(end = dimensions.paddingXSmall)
+                    .size(dimensions.paddingMedium),
+                imageVector = ImageVector.vectorResource(id = R.drawable.icon__check),
+                contentDescription = "Payment Fixed",
+                colorFilter = ColorFilter.tint(color = colors.textSecondary)
+            )
+        },
+        onClickAction = { onFixPaymentClicked.invoke() },
```

Comment:
I assume you didn't mean to add this click action here. I can remove it in my PR for MBL-1469 if you want